pkg/column-mapping/column.go

```diff
@@ -15,31 +15,59 @@ package column
 
 import (
 	"fmt"
+	"strconv"
 	"sync"
 
 	"github.com/juju/errors"
 	"github.com/pingcap/tidb-tools/pkg/table-rule-selector"
 )
 
+var (
+	// for partition ID, ref definition of partitionID
+	instanceIDBitSize       = 4
+	schemaIDBitSize         = 7
+	tableIDBitSize          = 8
```

Comment:
define them as `uint` ?

Reply:
it's ok, like `strconv.ParseUint`, bitsize is int